mediafile.py

```diff
@@ -1806,13 +1806,18 @@ def update(self, dict):
         MP3StorageStyle('TPE2'),
         MP4StorageStyle('aART'),
         StorageStyle('ALBUM ARTIST'),
+        StorageStyle('ALBUM_ARTIST'),
         StorageStyle('ALBUMARTIST'),
         ASFStorageStyle('WM/AlbumArtist'),
     )
     albumartists = ListMediaField(
         MP3ListDescStorageStyle(desc=u'ALBUMARTISTS'),
+        MP3ListDescStorageStyle(desc=u'ALBUM ARTISTS'),
+        MP3ListDescStorageStyle(desc=u'ALBUM_ARTISTS'),
         MP4ListStorageStyle('----:com.apple.iTunes:ALBUMARTISTS'),
         ListStorageStyle('ALBUMARTISTS'),
+        ListStorageStyle('ALBUM ARTISTS'),
+        ListStorageStyle('ALBUM_ARTISTS'),
         ASFStorageStyle('WM/AlbumArtists'),
```

Comment:
After adding those I'm getting concatenated arrays:
```
E   AssertionError: Tags did not match
E     Tag albumartists: ['multival', 'test', 'multival', 'test', 'multival', 'test'] != ['multival', 'test']
```

I'm not sure what's the best way to make it work, maybe someone have some idea?
Deduplicating this lists doesn't seem to be the best idea in my opinion and that's the only thing I thought personally.